error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1947,4 +1948,20 @@ Mono<T> after(CompletableFuture<T> future, boolean suppressCancel) {
       return Mono.fromFuture(() -> future, suppressCancel);
     }
   }
+
+  /**
+   * Prefer {@link Flux#fromStream(Supplier)} over {@link Flux#fromStream(Stream)}, as the former
+   * may defer initiation of the asynchronous computation until subscription.
+   */
+  static final class FluxFromStreamSupplier<T> {
+    @BeforeTemplate
```

Comment:
```suggestion
    // XXX: Constrain the `stream` parameter using `@NotMatches(IsIdentityOperation.class)` once
    // `IsIdentityOperation` no longer matches nullary method invocations.
    @BeforeTemplate
```
Is this `XXX` also relevant in this case?

Reply:
Yes, it is :+1: